Runtime/Audio/AudioManager.cs

```diff
@@ -0,0 +1,116 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+
+namespace Mixin.Utils.Audio
+{
+    /// <summary>
+    /// Singleton for managing the playing of various Audio Tracks and Playlists.
+    /// </summary>
+    public class AudioManager : Singleton<AudioManager>
+    {
+        private List<AudioTrackPlayer> _audioTrackPlayers = new List<AudioTrackPlayer>();
+        public List<AudioPlaylistPlayer> _audioPlaylistPlayers = new List<AudioPlaylistPlayer>();
```

Comment:
why is this public?
Please visit this guide about fields and properties:
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-properties